app/scripts/lib/strings.js

```diff
@@ -5,10 +5,14 @@
 define(function (require, exports, module) {
   'use strict';
 
+  const t = (msg) => msg;
 
-  var t = function (msg) {
-    return msg;
-  };
+  const HTML_CHAR_CODE_REGEXP = /&\D+;/i;
```

Comment:
This won't catch when the escape code is numeric. `&gt;` and `&#63;` are equivalent, this regex won't catch the latter.